src/test/java/testng/smoke/AddNewWidgetTest.java

```diff
@@ -0,0 +1,64 @@
+package testng.smoke;
+
+import com.report.portal.business.constants.WidgetType;
+import com.report.portal.business.steps.BaseSteps;
+import com.report.portal.business.steps.DashboardsSteps;
+import io.qameta.allure.Description;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static com.codeborne.selenide.Selenide.closeWebDriver;
+import static com.report.portal.business.constants.WidgetType.*;
+import static com.report.portal.business.steps.DashboardsSteps.createWidgetName;
+import static com.report.portal.core.utils.data.RandomSequenceCreator.generateRandomInt;
+
+public class AddNewWidgetTest {
+
+    private final BaseSteps baseSteps = new BaseSteps();
+    private DashboardsSteps dashboardsSteps;
+
+    private static final String DASHBOARD_NAME = "DEMO DASHBOARD";
+    private static final String FILTER_NAME = "DEMO_FILTER";
+
+    protected final int RANDOM_VALUE = generateRandomInt(1000, 9999);
+
+    @DataProvider(name = "widgetType", parallel = true)
+    public Object[][] getWidgetType() {
+        return new Object[][]{
+                {LAUNCH_STAT},
+                {LAUNCH_DURATION},
+                {OVERALL_STAT},
+                {FAILED_CASE_TREND}
+        };
+    }
+
+    @BeforeMethod
+    public void login() {
```

Comment:
Your test methods are just void. I suppose, BeforeMethod/AfterMethod annotated methods possibly could also be just void.

Reply:
fixed!